radio/util/tts_de.py

```diff
@@ -50,8 +50,8 @@
                 (u"Fehler", "error", NO_ALTERNATE),
                 (u"Trim zentriert", "midtrim", 495),
                 (u"Poti zentriert", "midpot", 496),
-                (u"Maximale Trimmung erreicht", "maxtrim", NO_ALTERNATE),
-                (u"Minimale Trimmung erreicht", "mintrim", NO_ALTERNATE),
```

Comment:
So it would be no problem if for those two files the same is spoken. Should i correct it?